.gitignore

```diff
@@ -4,7 +4,6 @@
 .DS_Store
 .rvmrc
 /coverage
-Gemfile.lock
 gemfiles/*.lock
```

Comment:
I kept appraisal lockfiles ignored because with the current Ruby version support bracket (2.6 ... 3.2) some dependencies like `nokogiri` cannot be locked due not having enough broad support ([1](https://rubygems.org/gems/nokogiri/versions/1.13.0-x86-linux), [2](https://rubygems.org/gems/nokogiri/versions/1.14.0-x86-linux)). Therefore [CI test matrix fails for some of this incompatible combinations](https://github.com/Betterment/delayed/actions/runs/7427493095/job/20213269793?pr=32).

Reply:
I'm not sure we get much of the benefits without this, but I'm approving this PR because at least the primary gemfile will allow for a consistent experience for someone who pulls down the repo.

Reply:
@smudge indeed, starting with the rubocop (lint job) not drifting due new cops available on each release (like the one I had to adjust myself here).